src/redux/news/news.sagas.js

```diff
@@ -67,10 +58,7 @@ function* handleArticleLoad({ payload }) {
 function* handleAddNews({ payload }) {
   try {
     yield put(setNewsLoading(true));
-    const result = yield call(createArticle, payload);
-    if (result.message) {
-      yield call(handleCustomNewsError, result.message[errorsLanguage].value);
-    }
+    yield call(createArticle, payload);
     const news = yield call(getAllNews, null);
```

Comment:
Why do we fetch again all the news after the creation of the article? It is a useless HTTP call. Maybe, it is much better to just take the result of mutation and dispatch it to the store. It will reduce quantity of HTTP requests: 2 requests => 1 request